data/clients.json

```
@@ -8,6 +8,17 @@
         ],
         "url": "https://adium.im"
     },
+    {
+        "doap": null,
+        "last_renewed": "2021-02-19T09:06:23",
+        "name": "Aparté",
+        "platforms": [
+            "Linux",
+            "macOS",
+            "*BSD",
```

```suggestion
            "BSD",
```

Continuous Integration fails. I suspect this is because the platform `*BSD` is not recognized. Try using `BSD` instead.

---

Thanks, first issue was the trailing comma and second one was the order.